openstack/glance/templates/etc/_glance-policy.json.tpl

```diff
@@ -1,6 +1,6 @@
 {
     "context_is_cloud_admin":  "role:cloud_image_admin",
-    "context_is_admin":  "rule:context_is_cloud_admin",
+    "context_is_admin":  "rule:context_is_cloud_admin or role:admin or role:image_admin",
```

Comment:
Why are you relaxing this rule?

Reply:
role:admin or role:image_admin can also set images visibility to community, these roles neednt add role:image_community_admin again.

Reply:
If I read the policy correctly they can't do this currently, rule `default` doesn't allow that.

Do you want to allow communitize_image for roles `admin` and `image_admin`, too? Please add it directly to that rule.